src/layman/layer/qgis/wms.py

```diff
@@ -88,9 +86,11 @@ def save_qgs_file(publ_uuid):
     ]
     source_type = util.get_source_type(db_types, qml_geometry)
     column_srid = db.get_column_srid(db_schema, table_name, table_uri.geo_column, uri_str=table_uri.db_uri_str)
-    layer_qml = util.fill_layer_template(layer, publ_uuid, layer_bbox, crs, qml, source_type, db_cols, table_uri,
-                                         column_srid, db_types)
-    qgs_str = util.fill_project_template(layer, publ_uuid, layer_qml, crs, settings.LAYMAN_OUTPUT_SRS_LIST,
+    layer_qml = util.fill_layer_template(layer.qgis_identification.name, layer.qgis_identification.id, layer_bbox, layer.native_crs, qml,
+                                         source_type, db_cols,
+                                         table_uri, column_srid, db_types)
```

Comment:
We can probably read some of these arguments from `layer` inside `fill_layer_template`.